src/lib/CollisionPrevention/collisionprevention_params.c

```diff
@@ -62,3 +62,15 @@ PARAM_DEFINE_FLOAT(MPC_COL_PREV_D, -1.0f);
  * @group Multicopter Position Control
  */
 PARAM_DEFINE_FLOAT(MPC_COL_PREV_DLY, 0.f);
+
+/**
+ * Angle at which the vehicle can fly away from obstacles
```

Comment:
@baumanta Could you try to explain this a bit better? Also, wouldn't it be better to define it in degrees here?